highfidelity/hifi_tests
# Set LOD for mac

# Description
1.  The automatically set LOD for Mac is not correct, causing tests to fail (the LOD oscillates, causing a matching oscillation in the scene.
2.  Simplify protocol tests.
3.  Updated recursive tests
# Testing
Mac LOD
======
1.  Run https://raw.githubusercontent.com/NissimHadar/hifi_tests/setLODForMac/tests/content/overlay/material/test.js
2.  Verify that no oscillations are observed.
Protocol tests
=========
1.  https://raw.githubusercontent.com/NissimHadar/hifi_tests/setLODForMac/tests/protocol/testRecursive.js
2.  Verify snapshot folder contains 11 passed text test results
Recursive scripts
===========
1.  Open the log window (CTRL+SHIFT+L)
2.  Run any  3 recursive scripts in https://raw.githubusercontent.com/NissimHadar/hifi_tests/setLODForMac/tests.
3.  Verify that all 3 have no critical errors in the log.
